website/content/docs/auth/gcp.mdx

```diff
@@ -346,22 +346,32 @@ Read more on the
 
 ### GCE
 
-GCE tokens **can only be generated from a GCE instance**. The JWT token can be
-obtained from the `service-accounts/default/identity` endpoint for a
-instance's metadata server.
+GCE tokens **can only be generated from a GCE instance**.
 
-#### curl Example
+1.  As of version 1.11, Vault can automatically discover the identity token on a GCE/GKE instance. This simplifies
```

Comment:
Hi, please consider removing calling out version 1.11 within the text. You can just start the sentence with "Vault can automatically discover the identify ...".

Reply:
Makes sense, I'll get it changed. Thanks!